thunder/dynamo/utils.py

```diff
@@ -176,7 +176,21 @@ def try_execute_thunder_symbol(thunder_symbol: Symbol, node: torch.fx.Node) -> t
     """
     import thunder
     from thunder.core.trace import TraceCtx
+    from thunder.core.compile_data import compile_data_and_stats
+    from thunder.common import CompileData, CompileStats
 
+    # We require a corresponding `_enter_autocast` in trace to correctly pop the state,
+    # else we get error notifying that we are popping empty stack.
+    # As a work-around, we short circuit and return True.
```

Comment:
Is there a better way around this? @IvanYashchuk do you have any idea or suggestion?

Reply:
I suggest adding `if not get_compile_data().autocast_stack: return` before `.pop()` call in
https://github.com/Lightning-AI/lightning-thunder/blob/c5f8bf73f21f20be8e748bbf0801d62e69f4bd92/thunder/torch/__init__.py#L5630-L5631